src/util/records.py

```diff
@@ -1210,6 +1223,17 @@ def add_ref(ref):
     if reset_write_metadata and write_data:
         cr.execute("UPDATE {} SET write_uid=%s, write_date=%s WHERE id=%s".format(table), write_data)
 
+    # force noupdate newly created record and inherit records
```

Comment:
I'm not sure what you meant here. But I think what you could do is to put the `noupdate` flag in `new_root` such that any new record created from it gets the noupdate flag.

Doing this post load may miss some newly created records (and children).